libs/core/src/lib/components/wrapper-component.ts

```diff
@@ -219,17 +218,17 @@ export abstract class ReactWrapperComponent<TProps extends {}> implements AfterV
       return (context: TContext) => htmlRenderer.render(context);
     }
 
-    if (input instanceof RenderOverride) {
-      return undefined;
-    }
-
     if (typeof input === 'object') {
-      const { componentType, factoryResolver, injector } = input;
-      const componentFactory = factoryResolver.resolveComponentFactory(componentType);
-      const componentRef = componentFactory.create(injector);
-
-      // Call the function again with the created ComponentRef<TContext>
-      return this.createInputJsxRenderer(componentRef, additionalProps);
+      if ('componentType' in input) {
+        const { componentType, factoryResolver, injector } = input;
+        const componentFactory = factoryResolver.resolveComponentFactory(componentType);
+        const componentRef = componentFactory.create(injector);
+
+        // Call the function again with the created ComponentRef<TContext>
+        return this.createInputJsxRenderer(componentRef, additionalProps);
+      } else {
+        return undefined;
```

Comment:
why is this `undefined` in the `DefaultRenderOptions` case?

Reply:
I understand the technical reason of. Why needing to handle this case is here - but why is returning undefined specifically? Shouldn't there be a render function call here, or rather, a function that creates a render function.

Reply:
Only because `unreachable(input)` throw an error during the compilation.
The execution never reach that line of code because the method `createRenderPropHandler` executes the cases RenderPropOptions and RenderPropContext.